<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form</class>
 <widget class="QDialog" name="form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>435</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите имя</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_login">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_mail">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_phone">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите номер телефона</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Зарегистрироваться </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_login">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите логин</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_password">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_phone">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_reg">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Регистрация</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mail">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите почту</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
